modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,275 @@
 
-* Runtime Fabric components are delivered via Anypoint Runtime Manager in Anypoint Platform. These components include the agent used to connect Runtime Fabric to Anypoint Platform. If an updated application or appliance version is available, a link to the downloads page is displayed in the *Upgrade Available* and *Appliance Version* field, as applicable.
+* Runtime Fabric components, which include the agent used to connect Runtime Fabric to Anypoint Platform.
++
+Updates are delivered via the Runtime Fabric page in Anypoint Runtime Manager. If an updated application or appliance version is available, a link to the downloads page is displayed in the *Upgrade Available* and *Appliance Version* fields, as applicable.
 
-* Node configuration changes are reflected in newer versions of the installer scripts, and are also delivered via the `rtfctl` command-line utility. These changes include adjusting operating system limits and performing jobs to clean up system resources.
+* The `rtfctl` command-line utility, which is used to manage Runtime Fabric and perform the actual upgrades to cluster-level and node configuration components.
++
+Upgrades are delivered as a new binary via the *Runtime Fabric downloads* page in Runtime Manager.
 
-* The `rtfctl` command-line utility upgrades are provided as a new binary that replaces the existing binary. The `rtfctl` utility is used to manage Runtime Fabric locally.
+* Node configuration, which includes scripts for provisioning AWS and Azure infrastructure, preparing each node, bootstrapping a new Runtime Fabric, adjusting operating system limits, and performing jobs to clean up system resources.
++
+Changes are delivered as an updated install script version via the *Runtime Fabric downloads* page in Runtime Manager.
++
+[NOTE]
+If you are upgrading the node configuration with install scripts, the latest version of the cluster-level components (installer package) is automatically downloaded and used.
 
-Upgrades to cluster-level and node configuration components are performed using the `rtfctl` command-line utility.
+It is important to run with the most current versions of the cluster-level (installer package) and `rtfctl` components.
 
-## Downloads page
+== Determine the Component Versions You Are Using
 
+* Run the following command:
++
+```
+/opt/anypoint/runtimefabric/rtfctl version
+```
+* The output provides the `rtfctl` version, the installer package version in the `appliance` field, and the Runtime Fabric version in the `agent` field, as shown in the following example:
++
+```
+[root@ip-10-0-245-74 /]# /opt/anypoint/runtimefabric/rtfctl version
+COMPONENT    VERSION                  
+rtfctl       0.2.96+2868e14           
+agent        1.5.4                    
+appliance    1.1.1581641679-5884bce   
+kubernetes   1.13.12
+```
++
+[NOTE]
+The Runtime Fabric version is also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, "Upgrade to vx.x" is displayed.
```

Comment:
"The Runtime Fabric component version and the appliance version are also displayed..."

Reply:
Updated - thank you!